metadata-ingestion/docs/sources/dummy-data/dummy-data_recipe.yml

```diff
@@ -0,0 +1,6 @@
+source:
+  type: dummy-data
```

Comment:
If we are not generalizing this, can we at minimum improve the name.

How about `demo-source` or `quickstart-data-source` or something similar. "dummy" doesn't have the best connotation overall

Reply:
demo-data?

Reply:
It was that in the beginning and @treff7es had asked me to change it to `dummy`